scripts/sdk/base-manifest.json

```diff
@@ -10,8 +10,6 @@
         },
         "./paths/vite": "./paths/vite.js",
         "./style.css": "./asset-build/assets/theme-element-light.css",
-        "./theme-element-light.css": "./asset-build/assets/theme-element-light.css",
```

Comment:
Could we keep these for backwards compatibility? Do keep the docs updated though.

Reply:
Added them back ⏩